src/js/view/view.js

```diff
@@ -415,8 +415,12 @@ define([
             this.checkResized();
         };
 
-        function itemReady() {
-            const title = _model.get('playlistItem').title || '';
+        function itemReady(item) {
+            const provider = _model.getVideo();
+            if (provider && provider.getName().name.indexOf('flash') === 0) {
```

Comment:
@karimJWP to @johnBartos' point, you can just compare `provider.getName().name === 'flash'` here. I'll merge once that's done.